### PR TITLE
Fix BE scraper

### LINKED ISSUE #324: BE scraper fails
[BE scraper fails](https://github.com/openZH/covid_19/actions/runs/66489667):

```
Run the scraper...
Error: Not matched input line: BE -                      -     826 FAILED 2020-03-30T12:16:55+00:00
# URLs: https://www.besondere-lage.sites.be.ch/besondere-lage_sites/de/index/corona/index.html
# Errors: Missing date. Missing cases. Date and time is None. Confirmed cases is None
```

### PR DESCRIPTION
**Pull request by amuelli**
Fixes #324

Adapted the BE scraper to the new html structure.
```
./scrape_be.sh
BE
Downloading: https://www.besondere-lage.sites.be.ch/besondere-lage_sites/de/index/corona/index.html
Scraped at: 2020-03-30T15:26:57+02:00
Date and time: 30.03.2020
Confirmed cases: 826
Hospitalized: 112
ICU: 21
Vent: 17
Deaths: 13
```

### CODE REVIEW

File: scrapers/scrape_be.sh [hunk 1]
```diff
@@ -8,32 +8,65 @@ sc.timestamp()
...
-print('Date and time:', sc.find(r'Stand: (.+)\)', d))
 # This isn't really the best method, but it works for now.
-print('Confirmed cases:', sc.find(r'<td .*<strong>([0-9]+)<', d))
-print('Deaths:', sc.find(r'<td[^<>]*>([0-9]+)</td>', d))
+print('Date and time:', sc.find(r'<td headers="th_top_5147_1A">(.+)<', d))
+print('Confirmed cases:', sc.find(r'<td headers="th_top_5147_2A">([0-9]+)<', d))
+print('Hospitalized:', sc.find(r'<td headers="th_top_5147_3A">([0-9]+)<', d))
+print('ICU:', sc.find(r'<td headers="th_top_5147_5A">([0-9]+)<', d))
+print('Vent:', sc.find(r'<td headers="th_top_5147_6A">([0-9]+)<', d))
+print('Deaths:', sc.find(r'<td headers="th_top_5147_7A">([0-9]+)<', d))
```
Comment (metaodi) [inline scrapers/scrape_be.sh]: I don't trust, that these ids like `th_top_5147_1A` are stable, so I wouldn't match exactly for them.

Note that there was another PR in the meantime for the same scraper: #329, it's matching seems a bit better, so I'll use their solution.

**Comment by metaodi** [comment]
Closing in favor of #329

**Comment by metaodi** [comment]
@amuelli sorry for this coincident of having two PRs for the same issue. Hope to see another contribution from you, as it is very much appreciated!

**Comment by baryluk** [comment]
@amuelli, thanks for trying to contribute and sending PRs quickly. Much appreciated! Hope to see your fixes for other code soon. :)

**Pull request closed without merging**